cmake/Linux.cmake

```diff
@@ -30,6 +30,10 @@ set(PNET_SNMP_PRIO 1
   CACHE STRING "SNMP thread priority")
 set(PNET_SNMP_STACK_SIZE 256*1024
   CACHE STRING "SNMP thread stack size")
+set(PNET_SNMP_OBJECT_ID NETSNMP_SYSTEM_MIB
```

Comment:
Are we sure about this? sysObjectID is defined as:
```

The vendor's authoritative identification of the
network management subsystem contained in the entity.
This value is allocated within the SMI enterprises
subtree (1.3.6.1.4.1) and provides an easy and
unambiguous means for determining `what kind of box' is
being managed. For example, if vendor `Flintstones,
Inc.' was assigned the subtree 1.3.6.1.4.1.424242,
it could assign the identifier 1.3.6.1.4.1.424242.1.1
to its `Fred Router'. 
```

By Profinet OID I assume you mean 1.3.6.1.4.1.24686, which is registered to PROFIBUS International. Is this not for products sold by PROFIBUS International or is it for all Profinet products?

Reply:
See above